src/app/components/home/home.component.html

```diff
@@ -21,7 +22,9 @@
             <mat-panel-title>
               <ng-container *ngIf="realm">
                 <div> {{realm.displayName}}</div>
-
+              </ng-container>
+              <ng-container *ngIf="!realm">
+                <div> Realm</div>
```

Comment:
in ce caz nu exista realm?

Reply:
La login

Reply:
Cand am avut (click)="gotoListItem,prima data realmul era null